server/app/jobs/dj_jobs/run_simulate_data_point.rb

```diff
@@ -476,24 +467,30 @@ def upload_file(filename, type, display_name = nil, content_type = nil)
     end
 
     def run_script_with_args script_name
-      dir_path = "#{analysis_dir}/scripts/data_point"
-      #  paths to check for args and script files
-      args_path = "#{dir_path}/#{script_name}.args"
-      script_path = "#{dir_path}/#{script_name}.sh"
-      log_path = "#{dir_path}/#{script_name}.log"
-
-      @sim_logger.info "Checking for presence of args file at #{args_path}"
-      args = nil
-      if File.file? args_path
-        args = Utility::Oss.load_args args_path
-        @sim_logger.info " args loaded from file #{args_path}: #{args}"
-      end
+      begin
+        dir_path = "#{analysis_dir}/scripts/data_point"
+        #  paths to check for args and script files
+        args_path = "#{dir_path}/#{script_name}.args"
+        script_path = "#{dir_path}/#{script_name}.sh"
+        log_path = "#{dir_path}/#{@data_point.id}.log"
```

Comment:
Would `"#{analysis_dir}/data_point_#{@data_point.id}/#{script_name}.log"` be workable?

Reply:
Actually `"#{analysis_dir}/data_point_#{@data_point.id}/run/#{script_name}.log"`

Reply:
I take it back, the first one is better.